tests/events/client/test_events_emit_event.py

```diff
@@ -35,7 +35,7 @@ def test_emits_complex_event(
     emit_event(
         event="vogon.poetry.read",
         resource={"prefect.resource.id": "vogon.poem.oh-freddled-gruntbuggly"},
-        occurred=DateTime(2023, 3, 1, 12, 39, 28, tzinfo=timezone.utc),
+        occurred=DateTime(2023, 3, 1, 12, 39, 28),
```

Comment:
It shouldn't have been necessary to pass a timezone here for the assertion below to see an event with a timezone